src/components/Switch.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import defaultTheme from '../themes/DefaultTheme';
+import darkTheme from '../themes/DarkTheme';
+
+const Switch = ({
+  label,
+  position,
+  checked,
+  disabled,
+  onChange,
+}) => (
+  <label
+    className={classnames('switch', {
+      [`switch--${position}`]: label && position,
+    })}
+  >
+    {label && <span className="switch__label">{label}</span>}
+    <input
+      className="switch__input"
+      type="checkbox"
+      checked={checked}
+      onChange={onChange}
+      disabled={disabled}
+    />
+    <div className="switch__button" />
+    <style jsx global>{`
+      .switch {
+        font-size: ${defaultTheme.font.size.normal};
+        font-family: ${defaultTheme.font.family.primary};
+        width: 100%;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        cursor: pointer;
+        user-select: none;
+      }
+
+      .switch--right {
+        flex-direction: row-reverse;
+      }
+
+      .switch--top {
```

Comment:
Right now, `top` and `bottom` are centered - I think we need to stick the to the left at the moment, and probably enhance the component that it can accept an object with positions